python/constants.py

```diff
@@ -16,11 +17,7 @@ class PyRasgoEnvironment(Enum):
 # Different Types of Transforms
 # This var is used to find all transforms of a type in the
 # directory <root>/<transform_type>_transforms/...
-TRANSFORM_TYPES = [
-    'column',
-    'row',
-    'table'
-]
+TRANSFORM_TYPES = ['column', 'row', 'table']
```

Comment:
@ChrisGriffithRASGO We used to use transform types for file directory structure but recently removed them and replaced it with transform 'tags'

We can delete this part of code if you don't see it used anywhere.

Reply:
Avoiding logic changes with this PR. Added as issue in linter. Good insight!